### PR TITLE
Bug 1215292 - Initial lint fixes

### PR DESCRIPTION
**Pull request by rwood-moz**

### CODE REVIEW

File: lib/device/lib/command.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ Command.prototype.adb = function(command) {
  * @returns {Command}
  */
 Command.prototype.adbShell = function(command) {
-  return this.adb("shell '" + command + "'");
+  return this.adb("shell '" + command + "'"); // eslint-disable-line
```
Comment (eliperelman) [inline lib/device/lib/command.js:56]: I'm assuming this is for double quotes. Once we move to ES6, we should remove this in favor of string template.

File: lib/device/lib/helpers.js [hunk 2]
```diff
@@ -237,7 +237,8 @@ Helpers.prototype.reboot = function() {
  */
 Helpers.prototype.restartB2G = function() {
   debug('Restarting B2G');
-  return this.executeWithDeviceTime("shell 'stop b2g && start b2g'");
+  //return this.executeWithDeviceTime("shell 'stop b2g && start b2g'");
```
Comment (eliperelman) [inline lib/device/lib/helpers.js]: Did we mean to remove this comment?

File: lib/device/lib/logging.js [hunk 3]
```diff
@@ -199,7 +198,8 @@ Logging.prototype.start = function() {
   if (device.properties['ro.build.version.sdk'] >= 21) {
     if (this.clearTimestamp) {
       args.push('-T');
-      args.push('"' + this.clearTimestamp.replace(/\r?\n|\r/, "") + '"');
+      //args.push('"' + this.clearTimestamp.replace(/\r?\n|\r/, "") + '"');
```
Comment (eliperelman) [inline lib/device/lib/logging.js]: Unused comment?

File: lib/device/lib/command.js (hunk 1 above)
Comment (rwood-moz) [inline lib/device/lib/command.js:56]: Correct

**Pull request merged**